test/utils.py

```diff
@@ -134,7 +134,8 @@ def expected_wheels(package_name, package_version, manylinux_versions=None,
         for platform_tag in platform_tags:
             wheels.append(f'{package_name}-{package_version}-{python_abi_tag}-{platform_tag}.whl')
 
-    if IS_WINDOWS_RUNNING_ON_TRAVIS:
+    custom_compiler = os.environ.get('DISTUTILS_USE_SDK') and os.environ.get('MSSdk')
+    if IS_WINDOWS_RUNNING_ON_TRAVIS and not custom_compiler:
```

Comment:
The Windows 2.7 newer compiler workaround now works on Travis. There's an issue somewhere where it was suggested that we allow this check to be loosened.